starlite/handlers.py

```diff
@@ -274,10 +276,8 @@ def __init__(
             self.http_method = HttpMethod.from_str(http_method)  # type: ignore
         if status_code:
             self.status_code = status_code
-        elif isinstance(self.http_method, list):
-            raise ImproperlyConfiguredException(
-                "When defining multiple methods for a given path, a status_code is required"
-            )
+        if isinstance(self.http_method, list):
+            self.status_code = HTTP_200_OK
```

Comment:
Whats the reason to change this behaviour?

Reply:
I had doubts about the original impl. for a while. I should've separated this to its own PR though.